js/src/ui/Form/AddressSelect/addressSelect.css

```diff
@@ -14,53 +14,110 @@
 /* You should have received a copy of the GNU General Public License
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
-.account {
-  padding: 0.25em 0;
-  display: flex;
-  align-items: center;
-}
 
-.name {
-  height: 32px;
-  line-height: 32px;
-  display: inline-block;
-  vertical-align: top;
-  text-transform: uppercase;
-  padding: 0 0 0 1em;
-}
+.input {
+  box-sizing: border-box;
+  appearance: textfield;
+  width: 100%;
+  padding: 0;
+  border: none;
+  background: transparent;
+
+  transition-property: font-size, padding;
+  transition-duration: 0.5s;
+  transition-timing-function: cubic-bezier(0.7,0,0.3,1);
+
+  color: white;
+  font-family: inherit;
+  font-size: 2em;
 
-.balance {
-  color: #aaa;
-  padding-left: 1em;
+  &:focus {
+    outline: none;
+  }
+
+  &::placeholder {
+    color: #a2a2a2;
+  }
 }
 
-.image {
-  display: inline-block;
-  height: 32px;
-  width: 32px;
-  margin: 0;
-  z-index: 10;
+.inputAddress {
+  position: relative;
+
+  &:hover, *:hover {
+    cursor: pointer !important;
+  }
 }
 
-.icon {
-  position: absolute;
+.main {
+  position: relative;
   left: 0;
-  top: 35px;
 
-  &.noLabel {
-    top: 11px;
+  &:focus {
+    outline: none;
   }
 }
 
-.paddedInput input {
-  padding-left: 46px !important;
+.label {
+  margin: 1rem 2rem 0.25em;
+  color: rgba(255, 255, 255, 0.498039);
 }
 
-.container {
+.underline {
   position: relative;
+  margin: 0 9rem 0 2rem;
+}
+
+.empty {
+  font-size: 1.5em;
+}
+
+.inputContainer {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+
+  .input {
+    font-size: 36px;
```

Comment:
Could we make it 1.5em (my mistake, the headings are 1.5em, not 35px - don't know where I got that number from)